pkgs/development/python-modules/tika-client/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+#, pytestCheckHook
+, pythonPackages
+}:
+buildPythonPackage rec {
+  pname = "tika-client";
+  version = "0.0.3";
+  format = "pyproject";
+
+  disabled = pythonOlder "3.8";
+
+  src = fetchFromGitHub {
+    owner = "stumpylog";
+    repo = "tika-client";
+    rev = version;
+    hash = "sha256-IKPTQ4n/j/W292F0JpSEUC0X8E1tr961WEcNCN5ymoU=";
+  };
+
+  propagatedBuildInputs = with pythonPackages; [
+    hatchling
+    httpx
+  ];
+  pythonImportsCheck = [
+    "tika_client"
+  ];
+  # These are the required test dependencies / config. But sicne 22 out of 23 tests
+  # require a tika Endpoint, they are disabled for now
+  #nativeCheckInputs = with pythonPackages; [
+  #  pytestCheckHook
+  #  magic
```

Comment:
Hint: This magic is not the required one for the tests.

Reply:
We require explicit test setups, since the implicit one uses `python setup.py test`, which is deprecated.

Reply:
Without magic, the tests failed for me due to missing imports. @mweinelt I'm a bit unsure on how to understand your remark. Should I change anything here or is it fine the way it is?

Reply:
> Hint: This magic is not the required one for the tests.

The correct package would be `python-magic`.

> I'm a bit unsure on how to understand your remark. Should I change anything here or is it fine the way it is?

Without a check hook the check phase runs `python setup.py test`, which is deprecated. So yes, I want you to set up `pytestCheckHook` to a reasonable degree, or disable the tests entirely by setting `doCheck = false; # <reason>`. Leaving it hanging like that is not acceptable.

Reply:
Thanks for elaborating. Due to the amount of failing (to their dependency on network / external services) I went with your proposal of just disabling all tests with a note on why.